stack_definition.json

```diff
@@ -186,222 +200,272 @@
-          "value": false
+          "name": "enable_platform_logs",
+          "value": "ref:../../inputs/enable_platform_logs_metrics"
+        },
+        {
+          "name": "enable_platform_metrics",
+          "value": "ref:../../inputs/enable_platform_logs_metrics"
         }
       ]
     },
     {
-      "name": "3 - Observability - Logging Monitoring Activity Tracker",
-      "version_locator": "7a4d68b4-cf8b-40cd-a3d1-f49aff526eb3.351cb33e-7037-4016-a38f-4e10c4a7ef68-global",
+      "name": "3b - Databases for Elasticsearch",
+      "version_locator": "7a4d68b4-cf8b-40cd-a3d1-f49aff526eb3.f4b9793a-611c-4f6c-9181-ca1fbdaa37fa-global",
       "inputs": [
         {
-          "name": "ibmcloud_api_key",
-          "value": ""
+          "name": "use_existing_resource_group",
+          "value": true
+        },
+        {
+          "name": "resource_group_name",
+          "value": "ref:../../members/1 - Account Infrastructure Base/outputs/workload_resource_group_name"
         },
         {
           "name": "region",
           "value": "ref:../../inputs/region"
         },
+        {
+          "name": "prefix",
+          "value": "ref:../../inputs/prefix"
+        },
+        {
+          "name": "plan",
+          "value": "ref:../../inputs/elasticsearch_plan"
+        },
+        {
+          "name": "elasticsearch_version",
+          "value": "8.12"
+        },
+        {
+          "name": "existing_kms_instance_crn",
+          "value": "ref:../../members/2a - Security Service - Key Management/outputs/kms_instance_crn"
+        },
+        {
+          "name": "kms_endpoint_type",
+          "value": "private"
+        },
+        {
+          "name": "member_host_flavor",
+          "value": "b3c.4x16.encrypted"
+        },
+        {
+          "name": "service_credential_names",
+          "value": {
+            "elastic_db_admin": "Administrator",
+            "wxasst_db_user": "Editor",
+            "toolchain_db_user": "Editor"
+          }
+        }
+      ]
+    },
+    {
+      "name": "4 - Event Notifications",
+      "version_locator": "7a4d68b4-cf8b-40cd-a3d1-f49aff526eb3.3910e11e-c90b-48b0-b271-ec5d5049ad84-global",
+      "inputs": [
+        {
+          "name": "kms_endpoint_url",
+          "value": "ref:../2a - Security Service - Key Management/outputs/kms_private_endpoint"
+        },
         {
           "name": "existing_kms_instance_crn",
           "value": "ref:../2a - Security Service - Key Management/outputs/kms_instance_crn"
         },
         {
-          "name": "resource_group_name",
-          "value": "ref:../1 - Account Infrastructure Base/outputs/observability_resource_group_name"
+          "name": "kms_endpoint_type",
+          "value": "private"
         },
         {
-          "name": "prefix",
-          "value": "ref:../../inputs/prefix"
+          "name": "resource_group_name",
+          "value": "ref:../1 - Account Infrastructure Base/outputs/workload_resource_group_name"
```

Comment:
I think this belongs to observability_resource_group_name (or may be the audit one), not the workload.

Reply:
used `observability_resource_group_name` RG